libs/gundeck-types/src/Gundeck/Types/Push/V2.hs

```diff
@@ -312,9 +312,9 @@ instance ToJSON Push where
         # "origin" .= _pushOrigin p
         # "connections" .= ifNot Set.null (_pushConnections p)
         # "origin_connection" .= _pushOriginConnection p
-        # "transient" .= ifNot (== False) (_pushTransient p)
-        # "native_include_origin" .= ifNot (== True) (_pushNativeIncludeOrigin p)
-        # "native_encrypt" .= ifNot (== True) (_pushNativeEncrypt p)
+        # "transient" .= ifNot not (_pushTransient p)
+        # "native_include_origin" .= ifNot id (_pushNativeIncludeOrigin p)
+        # "native_encrypt" .= ifNot id (_pushNativeEncrypt p)
```

Comment:
This doesn't seem like an improvement. `ifNot not` is pretty confusing. How about inverting the logic in `ifNot` and calling it something like `filterValue` or similar?
```haskell
filterValue f a = guard (f a) $> a
```
Just a suggestion.

Reply:
I'll have a look at it. Apply-refact did this but I agree it looks odd

Reply:
I have decided against this becuase `ifNot id` would the become `ifNot not` I honestly also cannot come up with a better name, `filter` is historically a really bad one :D